src/AutoRest.CSharp/Mgmt/Generation/ResourceOperationWriter.cs

```diff
@@ -83,6 +87,23 @@ private void WriteClientMethods(CodeWriter writer, ResourceOperation resourceOpe
                 writer.Line($"return await ListAvailableLocationsAsync(ResourceType, cancellationToken);");
             }
             writer.Line();
+
+            foreach (var item in context.Library.ChildParent)
```

Comment:
this should loop through context.Library.OperationGroups

If operationGroup.ParentResourceType == resourceOperation.OperationGroup.ResourceType

write the method.  You shouldn't need the parentchild thing you did

Reply:
Done